Separate browser extensions page (#2461)
privacyguides/privacyguides.org: docs/browser-extensions.md

---
title: Browser Extensions
icon: material/puzzle-outline
description: These browser extensions can enhance your browsing experience and protect your privacy.
cover: browser-extensions.webp
---

In general, we recommend keeping your browser extensions to a minimum to decrease your attack surface. They have privileged access within your browser, require you to trust the developer, can make you [stand out](https://en.wikipedia.org/wiki/Device_fingerprint#Browser_fingerprint), and [weaken](https://groups.google.com/a/chromium.org/g/chromium-extensions/c/0ei-UCHNm34/m/lDaXwQhzBAAJ) site isolation.

> However, some provide functionality which can outweighs these downsides in certain situations, particularly when it comes to [content blocking](basics/common-threats.md#mass-surveillance-programs).

Reviewer: ```suggestion
However, some provide functionality which can outweigh these downsides in certain situations, particularly when it comes to [content blocking](basics/common-threats.md#mass-surveillance-programs).
```

> Be particular about the extensions on this page you choose to install, and don't install anything which duplicates the functionality of another tool/extension you're already using, or don't immediately have a need for. For example, [Brave](desktop-browsers.md#brave) users don't need to install uBlock Origin, because Brave Shields already provides the same functionality.

Reviewer: ```suggestion
Be particular about the extensions on this page you choose to install. Don't install extensions which you don't immediately have a need for, or ones that duplicate the functionality of another tool/extension you're already using. For example, [Brave](desktop-browsers.md#brave) users don't need to install uBlock Origin, because Brave Shields already provides the same functionality.
```
I find the original syntax a little awkward (i.e., "don't install anything which [...] don't immediately have a need for").

Reviewer 2: I think we can do better here to be honest I don't really like "Be particular about the extensions on this page you choose to install" because what does that even mean?
